apps/webapp/app/v3/services/resumeAttempt.server.ts

```diff
@@ -24,7 +25,7 @@ export class ResumeAttemptService extends BaseService {
         },
         include: {
           taskRun: true,
-          taskRunDependency: {
+          dependencies: {
```

Comment:
@ericallam you need to look at this file. I don't have enough knowledge of the system right now.

Reply:
Previously, an attempt could only have a single dependency at a time. Meaning we had to clear it to add a subsequent one. (This is where we ran into unique constraint errors.)

The mechanics are still the same now, you can't wait for multiple dependencies at once. Only the latest dependency in the array is considered.

Reply:
I think this just needs testing on test cloud, it looks okay to me but hard to tell just by looking at it